print-apps/oereb/topicpage.jrxml

```diff
@@ -359,7 +359,7 @@
 				<textFieldExpression><![CDATA[$R{BBOXLegendLabel}]]></textFieldExpression>
 			</textField>
 			<subreport>
-				<reportElement stretchType="RelativeToTallestObject" mode="Transparent" x="193" y="2" width="299" height="16" uuid="f88a6ee0-42e5-4d2c-b1ab-9abb4f950664">
```

Comment:
Changing the y position of the sub-report placing will likely not solve the problem entirely but just reduces the symptoms. Looking at the actual sub-report it is larger than the space it gets in the report (see: https://github.com/openoereb/pyramid_oereb_mfp/blob/master/print-apps/oereb/topicotherlegend.jrxml#L3 to the right under `pageHeight`) This could be the actual cause of the overflow when the legend is larger.

Reply:
Following my calculation the value is 3, any other suggestions ?